spec/cohortManager.spec.js

```diff
@@ -0,0 +1,359 @@
+import CohortManager from '../src/cohortManager.js'
+import Cohort from '../src/cohort.js'
+import Student from '../src/student.js'
+
+describe('Cohort Manager:', () => {
+  let cohort
+
+  beforeEach(() => {
+    cohort = new CohortManager()
+  })
+
+  it('create cohort with cohort name', () => {
+    const expected = new Cohort('Cohort-1')
+
+    const result = cohort.createCohort('Cohort-1')
+
+    expect(result).toEqual(expected)
+  })
+
+  it("shouldn't create a cohort with same name as any other cohort", () => {
+    const expected = 'Cohort-1 already exists'
+    cohort.createCohort('Cohort-1')
+    const result = cohort.createCohort('Cohort-1')
+
+    expect(result).toEqual(expected)
```

Comment:
This can be difficult to work with as a consumer of your code. I'd want this function to throw an error if it can't do something rather than giving me a string that I need to programatically work with